ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm

```diff
@@ -702,37 +708,24 @@ Mobs that mostly focus on dealing RED damage, they are all a bit more frail than
 	if(grab_ready)
 		if(!istype(target, /mob/living/carbon/human))
 			return ..()
-		CounterGrab(target)
+		return CounterGrab(target)
 	return ..()
 
-/mob/living/simple_animal/hostile/lovetown/slumberer/CheckAndAttack() //stolen and modified from porccubus
-	if(!target)
-		return
-
-	var/distance = get_dist(target, src)
-	if(targets_from && isturf(targets_from.loc) && distance <= 2 && !incapacitated())
-		if (distance == 2 && !grab_ready)
-			return
-		AttackingTarget()
+/mob/living/simple_animal/hostile/lovetown/slumberer/OpenFire(target)
+	if(countering)
 		return
-
-/mob/living/simple_animal/hostile/lovetown/slumberer/RangedAttack(atom/A, params) //Ranged melee code for a client
-	if(!client)
-		return ..()
-	CheckAndAttack(A)
-	if(ismob(A))
-		changeNext_move(CLICK_CD_MELEE)
-	return ..()
+	if(get_dist(target, src) <= 2 && grab_ready)
+		return AttackingTarget()
```

Comment:
Did you find this resolved the issues for client-pilotted mbos as well?

Reply:
dodged it by removing reach, now client piloted mobs are correctly working too.